content/pl/dependencies.md

```diff
@@ -0,0 +1,12 @@
+## II. Dependencies
+### Jawnie zadeklaruj i wydziel zależności
+
+Większość języków programowania oferuje narzędzia do dystrybucji dodatkowych bibliotek, takie jak [CPAN](http://www.cpan.org/) dla Perla lub [Rubygems](http://rubygems.org/) dla języka Ruby. Biblioteki zainstalowane w ten sposób mogą być dostępne dla całego systemu (określane jako "site packages") lub ich zakres działania może być ograniczony dla pojedynczego projektu aplikacji (określane jako "vendoring" lub "bundling").
+
+**Aplikacja dwunastu aspektów nigdy nie powinna być zależna od bibliotek zainstalowanych dla całego systemu.** Wszystkie zależności są dokładnie określane przez dokument zawierający ich kompletną listę (*dependency declaration manifest*). Ponadto taka aplikacja korzysta z z narzędzia służącego do izolacji tych zależności podczas działania aplikacji, by upewnić się, że żadna z nich nie jest przypadkiem jedną z tych zainstalowanych w zewnętrznym środowisku, w którym jest uruchomiona. Pełna i dokładna specyfikacja, z których bibliotek korzysta aplikacja jest identyczna dla zarówno środowiska developerskiego jak i produkcyjnego.
```

Comment:
1. _że żadna z nich nie jest przypadkiem jedną z tych zainstalowanych w zewnętrznym środowisku, w którym jest uruchomiona_ źle to brzmi, spróbuj jakoś inaczej.
2. _Pełna i dokładna specyfikacja, z których bibliotek korzysta aplikacja jest identyczna dla zarówno środowiska developerskiego jak i produkcyjnego._ -> _Pełna i dokładna specyfikacja bibliotek używanych przez aplikację jest identyczna zarówno dla środowiska developerskiego jak i produkcyjnego_